buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -388,6 +388,14 @@
       "value": "false",
       "allowOverride": true
     },
+    "PB_ConfigurationGroup": {
+      "value": "Debug",
+      "allowOverride": true
+    },
+    "PB_Platform": {
```

Comment:
OK. nit but we should consider calling this pb_archgroup then to match the repo.